demos/simple-dist/index.js

```diff
@@ -1,42 +1,42 @@
 'use strict';
 
 function play () {
-  if (!window.iov) {
+  if (!window.clspIov) {
```

Comment:
@brainthinks I might want to wait for Bill's opinion. But if we are going to namespace it why not put it into a `clsp` object instead of a clsp prefix.

Reply:
i thought about doing that, but i decided not to to make it "easier" to pull what you need from the import statement (which was requested by the client) rather than making them import a single variable then destructure it later.  Also, in the examples I looked at, multiple exports was the norm - e.g. https://github.com/facebook/react/blob/master/packages/react/index.js